lib/mincer/engines/sass_engine.js

```diff
@@ -65,11 +65,16 @@ function sassError(ctx /*, options*/) {
 // Render data
 SassEngine.prototype.evaluate = function (context/*, locals*/) {
   try {
-    this.data = sass.renderSync({
+    var result = sass.renderSync({
       data:         this.data,
       includePaths: [ path.dirname(this.file) ].concat(context.environment.paths),
       indentedSyntax: /^.*\.sass$/.test(this.file)
     });
+    if (result.css) {
+      this.data = result.css;
+    } else {
+      this.data = result;
+    }
```

Comment:
I though of checking the version, but did not find a way to do this.
I could check sass.info (it's new in >v2.0.0). But it's the same as checking for result.css... :)